### PR TITLE
adding common name to tables

### PR DESCRIPTION
**Pull request by cmastr**
Signed-off-by: Casie Oxford <coxford@d2iq.com>

## Jira Ticket

https://jira.d2iq.com/browse/COPS-7149

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4390.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/release-notes/index.md [hunk 1]
```diff
@@ -71,39 +71,39 @@ This new predictive analytics tool provides greater support productivity, speed,
...
+| Istio                | istio                                          | 1.11.6  | - chart: 1.11.6 - istio: 1.11.5                                                                                        |
+| Jaeger               | jaeger                                         | 2.29.0  | - chart: 2.29.0 - jaeger: 1.31.0                                                                                       |
+| Karma                | karma                                          | 2.0.1   | - chart: 2.0.1 - karma: 0.88                                                                                           |
+| Kiali                | kiali                                          | 1.47.0  | - chart: 1.47.0 - kiali: 1.47.0                                                                                        |
+| KNative              | knative                                        | 0.3.9   | - chart: 0.3.9 - knative: 0.22.3                                                                                       |
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/release-notes/index.md:87]: I have seen this way of writing it several times, but when I google the app, it seems like it is written with lower-case?
```suggestion
| KNative              | knative                                        | 0.3.9   | - chart: 0.3.9 - knative: 0.22.3                                                                                       |
```

**Review by gvasquezvargas — APPROVED** [review]
Added the words that were giving us Vale errors to the accept file + 1 minor comment.

**Comment by gracedo** [comment]
sorry, merging my PR created merge conflicts here. I'll resolve

File: pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ This section only details requirements for management cluster-specific applicati
...
-| dex-k8s-authenticator     | cpu: 100m<br />memory: 128Mi   |                                     |
-| gitea                     | cpu: 500m<br />memory: 512Mi   | # of PVs: 2<br />PV sizes: 10Gi     |
-| karma                     |                                |                                     |
-| kommander-flux            | cpu: 4000m<br />memory: 4Gi    |                                     |
-| kubefed                   | cpu: 300m<br />memory: 192Mi   |                                     |
-| thanos                    |                                |                                     |
-| traefik-forward-auth-mgmt | cpu: 100m<br />memory: 128Mi   |                                     |
+| Common Name       | Application Name          | Minimum Resources Suggested | Minimum Persistent Storage Required |
+|-------------------|---------------------------|-----------------------------|-------------------------------------|
+| Grafana           | centralized-grafana       | cpu: 200m memory: 100Mi     |                                     |
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md:16]: We need to keep the `<br>` tags for the line breaks. I'll push a commit to add these back

File: pages/dkp/kommander/2.0/workspaces/workspace-platform-services/index.md [hunk 3]
```diff
@@ -28,32 +28,30 @@ To use the CLI to deploy or uninstall applications, see [Application Deployment]
...
-| metallb-0.12.2                | metallb               | False               |
-| minio-operator-4.1.7          | minio-operator        | False               |
-| nvidia-0.4.2                  | nvidia                | False               |
-| prometheus-adapter-2.11.1     | prometheus-adapter    | True                |
-| reloader-0.0.99               | reloader              | True                |
-| traefik-10.3.0                | traefik               | True                |
-| traefik-forward-auth-0.3.2    | traefik-forward-auth  | True                |
-| velero-3.1.3                  | velero                | False               |
-
+| Common Name          | Application Version           | APP ID                | Deployed by default |
```
Comment (gracedo) [inline pages/dkp/kommander/2.0/workspaces/workspace-platform-services/index.md:31]: this is updating a 2.0 page, we probably want these on 2.1 and 2.2 as well?

File: pages/dkp/konvoy/2.2/release-notes/index.md [hunk 4]
```diff
@@ -71,39 +71,39 @@ This new predictive analytics tool provides greater support productivity, speed,
...
-| project-grafana-logging| 6.20.6 | - chart: 6.20.6<br>- grafana: 8.3.6 |
-| project-grafana-loki | 0.33.2 | - chart: 0.33.1<br>- loki: 2.2.1 |
-| project-logging | 1.0.0 |  |
-| prometheus-adapter | 2.17.1 | - chart: 2.17.1<br>- prometheus-adapter: 0.9.1 |
-| reloader | 0.0.104 | - chart: 0.0.104<br>- reloader: 0.0.104 |
-| thanos | 0.4.6 | - chart: 0.4.6<br>- thanos: 0.9.0 |
-| traefik | 10.9.1 | - chart: 10.9.1<br>- traefik: 2.5.6 |
-| traefik-forward-auth | 0.3.6 | - chart: 0.3.6<br>- traefik-forward-auth: 3.1.0 |
-| velero | 3.2.0 | - chart: 3.2.0<br>- velero: 1.5.2 |
+| Application Name     | Platform Application                           | Version | Component Versions                                                                                                     |
```
Comment (gracedo) [inline pages/dkp/konvoy/2.2/release-notes/index.md:74]: we probably need to make the same changes on kommander 2.2 release notes as well?

File: pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md (hunk 2 above)
Comment (gracedo) [reply pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md]: done in https://github.com/mesosphere/dcos-docs-site/pull/4390/commits/2552f07c2c0239f13bec16605292ff80267e02d7

File: pages/dkp/kommander/2.0/workspaces/workspace-platform-services/index.md (hunk 3 above)
Comment (gracedo) [reply pages/dkp/kommander/2.0/workspaces/workspace-platform-services/index.md]: done in https://github.com/mesosphere/dcos-docs-site/pull/4390/commits/867e27fa84a097576d0895fd9d84014881a946fc

**Review by gracedo — APPROVED** [review]

**Comment by gvasquezvargas** [comment]
I somehow don't see the changes reflected in the preview for pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md and pages/dkp/kommander/2.2/release-notes/2.2.0/index.md.

**Review by gflorinskaya — APPROVED** [review]

**Comment by mmcelroy91** [comment]
Gabby, I'm not seeing them either.... going to make one change and commit again to see if Jenkins was the problem from yesterday.

**Pull request merged**